clients/apps/web/next.config.js

```diff
@@ -91,6 +98,54 @@ const nextConfig = {
         permanent: false,
       },
 
+      // Logged-out user redirection
+      {
+        // Paths that shall remain public
+        source: '/((?!_next/static|_next/image|favicon.ico|favicon.png|login).*)',
```

Comment:
This is great! Really nice idea!

There is a few more pages that can handle logged out users:

* `/maintainer/signup`
* `/github/installation` (supports both logged in and logged out users)
* `/github/session`

and maybe even a few more? What about the magic link pages?

Reply:
Also all `/api/`, `.well-known/`, `/embed/`, and `/og/` routes

Reply:
`/new/`

Reply:
And all public user pages, the pledge pages, etc.

Reply:
It might actually be easier to add a list of pages here that are for logged in users only than doing the opposite. That way we don't need to list all pages in the route, as we would handle it client side of we miss to configure a page.

Reply:
Good point 😄 I didn't think we had so many public pages!

Reply:
It was more than I thought too! :D

Reply:
Much cleaner like this indeed!